src/sdkV2.ts

```diff
@@ -1886,43 +1886,24 @@ export default class CegaEvmSDKV2 {
   }
 
   async sfnBulkEndAuctions(
-    vaultAddresses: EvmAddress[],
-    auctionWinners: EvmAddress[],
-    tradeStartDates: Date[],
-    lendingAprBps: number[],
-    maxAprBps: number[],
-    oracleDataSources: OracleDataSource[][],
+    endAuctionParams: SFNEndAuctionParam[],
     overrides: TxOverrides = {},
   ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = await this.loadCegaEntry();
-    const tradesStartInSeconds = tradeStartDates.map((tradeStartDate) =>
-      Math.floor(tradeStartDate.getTime() / 1000),
-    );
+    const params = endAuctionParams.map((param) => ({
+      ...param,
+      tradeStartDates: Math.floor(param.tradeStartDate.getTime() / 1000),
```

Comment:
can you double check?
`tradeStartDate` i think should be with out the `s`

Reply:
@vsai thanks, changed it, also added a type interface just to avoid this kind of mistake.